app/src/main/java/org/oppia/app/help/faq/FAQActivityPresenter.kt

```diff
@@ -10,9 +12,17 @@ import javax.inject.Inject
 class FAQActivityPresenter @Inject constructor(private val activity: AppCompatActivity) {
   fun handleOnCreate() {
     activity.setContentView(R.layout.faq_activity)
+    setUpToolbar()
     activity.supportFragmentManager.beginTransaction().add(
```

Comment:
Always check if the fragment is already present or not before adding it.

Check the implementation in `MyDownloadsActivityPresenter` where we first check if fragment is present or not. Also, we need to fix this same thing in `HelpActivityPresenter` also.

Reply:
done